src/models/DFSPEndpointItemModel.js

```diff
@@ -111,11 +114,11 @@ exports.findAllDfspState = async (dfspId, state) => {
  */
 exports.findObjectByDirectionType = async (direction, type, dfspId) => {
   const id = await DFSPModel.findIdByDfspId(dfspId);
-  const rawObjects = await knex.table(ENDPOINT_ITEMS_TABLE)
+  const rawObjects = await runQuery(() => dbTable
     .where(`${ENDPOINT_ITEMS_TABLE}.dfsp_id`, id)
     .where(`${ENDPOINT_ITEMS_TABLE}.direction`, direction)
     .where(`${ENDPOINT_ITEMS_TABLE}.type`, type)
-    .select(`${ENDPOINT_ITEMS_TABLE}.*`);
+    .select(`${ENDPOINT_ITEMS_TABLE}.*`));
```

Comment:
Maybe pass the function name as value for the `operator` argument here and everywhere else.